src/dwarf/constants.rs

```diff
@@ -0,0 +1,124 @@
+#[allow(dead_code, non_upper_case_globals)]
+pub const DW_UT_compile: u8 = 0x1;
```

Comment:
do we use gimli-rs? If yes, why don't we just use constants from DW_TAG_array_type. If not, why not?

Reply:
I'd also want to understand why we don't use existing crates, for dwarf but also ELF stuff.